DataFormats/EgammaCandidates/src/GsfElectron.cc

```diff
@@ -71,6 +71,29 @@ GsfElectron::GsfElectron
  }
 
 GsfElectron::GsfElectron
+ ( int charge, const ChargeInfo & chargeInfo,
+   const GsfElectronCoreRef & core,
+   const TrackClusterMatching & tcm, const TrackExtrapolations & te,
+   const ClosestCtfTrack & ctfInfo,
```

Comment:
.. actually, do you even need to define this new constructor?
Can the old one be extended with one more argument?